schemas/com.roku/device_info/jsonschema/1-0-0

```diff
@@ -0,0 +1,159 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Context Schema with information about Roku device (reflects the ifDeviceInfo interface: https://developer.roku.com/en-gb/docs/references/brightscript/interfaces/ifdeviceinfo.md)",
+  "self": {
+    "format": "jsonschema",
+    "name": "device_info",
+    "vendor": "com.roku",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "model": {
+      "type": "string",
+      "description": "Model name of the Roku device.",
+      "maxLength": 255
+    },
+    "modelDisplayName": {
+      "type": "string",
+      "description": "Model display name.",
+      "maxLength": 65535
+    },
+    "modelType": {
```

Comment:
Without having looked at the reflected spec, should this be an enum?

Reply:
I think you're right, ideally it should be an enum. However, Roku releases new device models quite often – won't it be a problem that the schema will need to be updated for each new model? We would have to release new schema versions and also update the tracker to use the new schema versions.

Reply:
Ideally yes, it should be an enum. But it's not actually under our control – the value is taken from the Roku SDK and they can add or change the values in future releases. Because of that, I am a bit wary of listing explicit values in the schema. What do you think?

Reply:
This equally makes me nervous. It's not even a particularly well defined and peer reviewed spec like a web standard might be, we could easily end up in a situation with failing events that are completely out of our control.